core/src/main/java/com/spotify/metrics/core/SemanticMetricRegistry.java

```diff
@@ -47,20 +48,21 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.function.Supplier;
 
 /**
  * A registry of metric instances.
  */
 public class SemanticMetricRegistry implements SemanticMetricSet {
     private final ConcurrentMap<MetricId, Metric> metrics;
     private final List<SemanticMetricRegistryListener> listeners;
+    private final Supplier<Reservoir> defaultReservoirSupplier;
 
     /**
      * Creates a new {@link SemanticMetricRegistry}.
      */
     public SemanticMetricRegistry(final ConcurrentMap<MetricId, Metric> metrics) {
-        this.metrics = metrics;
-        this.listeners = new CopyOnWriteArrayList<SemanticMetricRegistryListener>();
```

Comment:
What's the reasoning for removing this listener?

Reply:
Not really removed, just moved to a shared constructor

Reply:
thanks missed that 👍